### ISSUE TITLE
Important Feature which can be added to the next update

### ISSUE THREAD

**Issue by vishesh0311** [ukanth/afwall#530]
Sir,

AFWall+ is one of the greatest apps I have come through and I appreciate your efforts. My only concern is that whenever a new application is installed it asks me whether to give wifi/data access to the app.  If you could provide the feature where I can set that the newly installed app by default gets WiFi access but not data access or vice versa or gets access to both in the whitelist mode, I will be very grateful. This is one of the most important feature and I hope that you will provide this feature to set the default access for the newly installed app in your next update.

Thanks

**Comment by Gitoffthelawn** [comment]
Interesting.  Why would you like it to have this feature?

**Comment by vishesh0311** [comment]
Because whenever I install a new app it everytime asks me to configure the internet permission for that app. And I want that the newly installed app should get Wifi access but not Data Access automatically without needing me to open AFwall+ everytime, then click on tick below Wifi and then Apply. Btw I am in whitelist mode. Thatswhy I want that it should provide an option to choose by default that the new app should get this access or this access automatically and apply it. Thanks

**Comment by CHEF-KOCH** [comment]
Would it not be easier to globally set Wifi only and only allow cellular networks for e.g. WhatsApp? This already can be done within Android. Some apps also detect this automatically and some also offering special settings for this. There exist also [apps for such a case](https://play.google.com/store/apps/details?id=com.onavo.android.onavoid).

**Comment by vishesh0311** [comment]
Actually what you are saying is not at all related to my concern. I know I can set wifi for global usage but there are some apps which I don't want wifi to use that's why I am operating in whitelist mode. If you could  add this feature in next update it can lower my issues.. Thanks

**Comment by CHEF-KOCH** [comment]
Sounds okay, you mean a whitelist for specific interface based on per-app basis.

**Comment by vishesh0311** [comment]
No, I am saying that in whitelist mode add an option where user can set the usage of Wifi/Data by default..so that when new app gets installed it grants the default permission to the app set by the user.. For example, If I am operating in whitelist mode and I will set this option by default that the newly installed app should get only wifi access and not data access so whenever the new app installs it should automatically tick mark on Wifi access but not on Data access and apply the tables automatically without asking the user anything.. I hope now u would have understood clearly my concern. And I will be very grateful if you could add this feature in your next update. Thanks

**Comment by someone-somenet-org** [comment]
Sounds counter-security and would therefore be a bad idea.
